responders/src/API/EMBC.Responders.API/Controllers/SuppliersController.cs

```diff
@@ -119,12 +120,13 @@ public class SuppliersMapping : Profile
         public SuppliersMapping()
         {
             CreateMap<ESS.Shared.Contracts.Suppliers.Supplier, Supplier>()
-                .ForMember(d => d.MutualAid, opts => opts.MapFrom(s => !s.IsPrimarySupplier))
+                .ForMember(d => d.IsPrimarySupplier, opts => opts.MapFrom(s => s.IsPrimarySupplier))
```

Comment:
I assume these changes can be rolled back until after the new shared contract is published?